chapters/VK_KHR_swapchain/wsi.txt

```diff
@@ -158,9 +158,14 @@ The native window referred to by pname:pCreateInfo->surface must:
 not become associated with a non-Vulkan graphics API surface before
 all associated Vulkan swapchains have been destroyed.
 
-Like core functions, several WSI functions, including
-fname:vkCreateSwapchainKHR return ename:VK_ERROR_DEVICE_LOST if the logical
-device was lost.
+fname:vkCreateSwapchainKHR can return ename:VK_ERROR_DEVICE_LOST if the logical
+device was lost. The sname:VkSwapchainKHR is a child of the device and
+is affected by the lost state; it must: be destroyed before destroying the
+sname:VkDevice. However, sname:VkSurfaceKHR is not a child of any sname:VkDevice
+and is not otherwise affected by the lost device.
```

Comment:
What does the "otherwise" refer to? How is the surface affected by the lost device?

Reply:
removed